orbit/initializer/lgt.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+from ..constants import lgt as constants
+
+
+class LGTInitializer(object):
```

Comment:
Since `LGTInitializer` and `DLTInitializer` are identical, can they just be a single class?

Reply:
I thought about that but theoretically they can be different process.  It just LGT and DLT share similar process for now.  And unfortunately ETS are not doing regression.  I am still open to putting them together as ETSInitializer but i think conceptually they are not necessary the same and can split in the future.